drake_ros/CMakeLists.txt

```diff
@@ -19,10 +19,17 @@ find_package(pybind11 REQUIRED HINTS "${drake_DIR}/../pybind11" NO_DEFAULT_PATH)
 add_subdirectory(core)
 add_subdirectory(tf2)
 add_subdirectory(viz)
-# Python bindings.
+# Python bindings
 add_subdirectory(drake_ros)
 
+# Install python package must happen at top level CMakeLists.txt
```

Comment:
Not sure if I understand this part - why must it happen at top-level, but included subdirectory has an install directive for Python stuff?

Reply:
The `ament_python_install_package()` call has to be in the top level `CMakeLists.txt`. It sets variables in the parent scope which are picked up by a hook in `ament_package()`. `add_subdirectory()` creates a new scope in CMake, so `ament_python_install_package()` can't be put into it as `ament_package()` wouldn't see the variables being set.

Reply:
Gotcha. In that case, can you qualify the comment, e.g.

```
# Ament installation must happen in top-level, including Python package installation
```